static/test/javascripts-legacy/spec/commercial/messenger/resize.spec.js

```diff
@@ -1,12 +1,13 @@
 define([
```

Comment:
It is difficult to convert `should accept all relative units` test to Jest. I spent all of yesterday fixing inconsistencies between the legacy and new test, only to immediately run into the next one. 😄 🔫 

For some reason, Jest will not set non-px heights on html elements, while the legacy tests would. I've decided to make the `should accept all relative units` integration test a unit test until I can figure something out...